gstools/covmodel/models.py

```diff
@@ -433,6 +435,36 @@ def calc_integral_scale(self):  # noqa: D102
         )
 
 
+class Cubic(CovModel):
+    r"""The Cubic covariance model.
+
+    A model with reverse curvature near the origin and a finite range of
+    correlation.
+
+    Notes
+    -----
+    This model is given by the following correlation function:
+
+    .. math::
+       \rho(r) =
+       \begin{cases}
+       1- 7 \left(s\cdot\frac{r}{\ell}\right)^{2}
+       + \frac{35}{4} \left(s\cdot\frac{r}{\ell}\right)^{3}
+       - \frac{7}{2} \left(s\cdot\frac{r}{\ell}\right)^{5}
+       + \frac{3}{4} \left(s\cdot\frac{r}{\ell}\right)^{7}
+       & r<\frac{\ell}{s}\\
+       0 & r\geq\frac{\ell}{s}
+       \end{cases}
```

Comment:
If I read this correctly, it's the same implementation I used.

Reply:
Think so, too! ;-)